FPPicker/Platforms/iOS/FPLocalAlbumController.m

```diff
@@ -103,6 +103,21 @@ - (void)loadAlbumData
     fetchOptions.predicate = [NSPredicate predicateWithFormat:predicateFormat
                                                 argumentArray:arguments];
 
+
+    // Fix #104: Request authorization for the Photo Library, if the app has not already done so
+    if ([PHPhotoLibrary authorizationStatus] == PHAuthorizationStatusNotDetermined) {
+        [PHPhotoLibrary requestAuthorization:^(PHAuthorizationStatus status) {
+            dispatch_async(dispatch_get_main_queue(), ^{
+                // Retry loading the images. Dispatch onto the main thread, since this block
+                // will be called from a background thread.
+                [self loadAlbumData];
+            });
+        }];
+
+        // Wait until after the user has responded to the prompt.
+        return;
```

Comment:
I tested and it works well, but it will be more clearly when we move this code to viewWillAppear: method.